scripts/load-environment-from-secrets.sh

```diff
@@ -5,10 +5,15 @@
   "AWS_SECRET_ACCESS_KEY" \
   "AWS_ACCESS_KEY_ID"
 
-REGION=us-east-1
-content=$(aws secretsmanager get-secret-value --region ${REGION} --secret-id "${ENV}_deploy" --query "SecretString" --output text)
-echo $content
-echo ${content} | jq -r 'to_entries|map("\(.key)=\"\(.value)\"")|.[]' > .env
-set -o allexport
-source .env
-set +o allexport
\ No newline at end of file
+if [ $? = 0 ]; then
```

Comment:
Does this work?  I thought for numbers it needs to be `-eq`, not `=`.

Reply:
You were right. Thanks!